parachain/runtime/src/lib.rs

```diff
@@ -372,10 +382,10 @@ impl pallet_democracy::Config for Runtime {
     /// A straight majority of the council can decide what their next motion is.
     type ExternalOrigin = EnsureRootOrHalfGeneralCouncil;
     /// A majority can have the next scheduled referendum be a straight majority-carries vote.
-    type ExternalMajorityOrigin = EnsureRootOrHalfGeneralCouncil;
```

Comment:
This increases the external majority to the entire council when the default was half the council. Not sure why, can you let me know?